DatabaseHandler/DB_AccController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DatabaseHandler
+{
+    public class DB_AccController
```

Comment:
Can this have a normal C# class name? `AccountController` seems fine to me. Others might have a different opinion for the naming convention but not sure `DB_AccController` is right